src/resolvers/noteResolver.integration.test.ts

```diff
@@ -0,0 +1,122 @@
+import createTestClient from 'supertest';
+import { createConnection, getConnection } from 'typeorm';
+import { getExpressServer } from '../express-server';
+import Note from '../models/Note';
+import User from '../models/User';
+import Role from '../models/Role';
+import Language from '../models/Language';
+import Message from '../models/Message';
+import Appointement from '../models/Appointement';
+import UserSession from '../models/UserSession';
+
+describe('Note resolvers', () => {
+  let testClient: createTestClient.SuperTest<createTestClient.Test>;
+  beforeEach(async () => {
+    await createConnection({
```

Comment:
La création de la connexion pourrait être mise dans une fonction dans un fichier à part, réutilisable dans les différents tests, non ?

Reply:
Oui, je suis d'accord, je viens de la créer mais j'ai du mal avec le type que doit retourner la fonction. Car le type de base ne supporte pas l'async

Reply:
Ce n'est pas `Promise<…>` ?

Reply:
http://gph.is/1BAylu9